sdk/analytics/src/main/java/com/klaviyo/analytics/state/PersistentObservableString.kt

```diff
@@ -1,11 +1,11 @@
 package com.klaviyo.analytics.state
 
-import com.klaviyo.analytics.model.ProfileKey
+import com.klaviyo.analytics.model.Keyword
 import com.klaviyo.core.Registry
 import kotlin.reflect.KProperty
 
 internal class PersistentObservableString(
-    key: ProfileKey,
+    key: Keyword,
```

Comment:
this is why i originally had the API_KEY subclass from ProfileKey, seems like changing this type should be fine but lmk if you feel otherwise

Reply:
Ooh I see, no its better if this is `Keyword`!